dora/core/server/worker/src/main/java/alluxio/worker/netty/PacketReadTask.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.netty;
+
+import io.netty.channel.Channel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.Callable;
+
+/**
+ * A packet read task represents a task for reading packet according to request from Netty Client.
+ */
+public class PacketReadTask implements Callable<Boolean> {
+
+  private static final Logger LOG = LoggerFactory.getLogger(PacketReadTask.class);
+
+  private String mTaskId;
+
+  private PacketReadTaskStateMachine mStateMachine;
+
+  /**
+   * Creates an instance of the {@link PacketReadTask}.
+   *
+   * @param taskId the identifier for the packet read task
+   * @param context context of the request to complete
+   * @param channel the channel
+   * @param packetReader the packet reader for reading packet
+   */
+  public PacketReadTask(String taskId, ReadRequestContext context, Channel channel,
+                        AbstractReadHandler.PacketReader packetReader) {
```

Comment:
IDE is warning about raw use of parametric types. Use `ReadRequestContext<?>` and `AbstractReadHandler<?>`.